cbw_api_toolbox/cbw_api.py

```diff
@@ -45,15 +47,23 @@ def _request(self, route, params=None):
             self.logger.error("An error occurred, please check your API_URL.")
             os._exit(-1)
 
+    # GET request to /api/v2/ping then check uuid value
+
     def ping(self):
-        result = self._request(ROUTE_GET_PING)
+        result = self._request(ROUTE_PING, "GET")
         print("OK") if result and 'uuid' in result else self.logger.error("FAILED")
 
+    # GET request to /api/v2/servers to get all servers
```

Comment:
The comment should be right above the method definition

Reply:
Done